core/templates/dev/head/css/oppia.css

```diff
@@ -3187,6 +3187,20 @@ md-card.preview-conversation-skin-supplemental-card {
   color: blue;
 }
 
+.tile-tooltip .tooltip.right {
```

Comment:
Can you move these three classes into exploration_summary_tile_directive.html, since they're only used there? Have a look at other directives that have their own CSS to see how this is done.

Also I suggest prefixing these with .oppia-activity-summary-tile (or the custom tag name) instead. It's conceivable that other parts of the app may use custom things like tile-tooltip.